DuoSecurity.Auth.Http/JsonModels/DeviceModel.cs

```diff
@@ -8,7 +8,19 @@ namespace DuoSecurity.Auth.Http.JsonModels
 {
     internal class DeviceModel
     {
-        public IEnumerable<string> Capabilities { get; set; }
+        private IEnumerable<string> _capabilities;
+
+        public IEnumerable<string> Capabilities
+        {
+            get
+            {
+                if (_capabilities == null)
+                    _capabilities = new List<string>();
```

Comment:
I think this would do better as `Enumerable.Empty<string>()` as opposed to `new List<string>()`. I think there is some additional (probably negligible) overhead with making a `List<T>`, but like I said a minor nitpick.